src/common.h

```diff
@@ -86,8 +86,11 @@
  * Change the visibility of interface functions
  */
 
-#define LC3_EXPORT \
-    __attribute__((visibility ("default")))
+#if _MSC_VER
```

Comment:
You'll trigger  `-Wundef` on non-Microsoft compilers. Since both Visual Studio and MinGW/MSYS2 support `__declspec(dllexport)`, I propose:

```suggestion
#ifdef _WIN32
```